.github/actions/turbopack-bump/action.yml

```diff
@@ -0,0 +1,5 @@
+name: "turbopack-bump"
```

Comment:
Should the action.yml list the `inputs` and `outputs`

Reply:
Good catch, done.